.github/workflows/JuliaFormatter.yml

```diff
@@ -35,7 +31,7 @@ jobs:
     - uses: julia-actions/setup-julia@latest
       if: steps.filter.outputs.julia_file_change == 'true'
       with:
-        version: 1.5.2
+        version: 1.5.3
```

Comment:
Sorry I have a few questions. Is this version change needed to fix the current formatter issue? Should we communicate it? Should people also use Julia 1.5.3 locally?

Reply:
no

Reply:
ah! Thanks for clarifying